tests/app/service/send_notification/test_send_notification.py

```diff
@@ -1022,8 +1031,8 @@ def test_should_send_notification_to_guest_list_recipient(
 @pytest.mark.parametrize(
     "notification_type, template_type, to",
     [
-        (EMAIL_TYPE, SMS_TYPE, "notify@digital.fake.gov"),
-        (SMS_TYPE, EMAIL_TYPE, "+12028675309"),
+        (NotificationType.EMAIL, TemplateType.SMS, "notify@digital.fake.gov"),
+        (NotificationType.SMS, TemplateType.EMAIL, "+12028675309"),
```

Comment:
Calling Jenny... except we don't want to 😆 

(flagging another test email/phone number combo)